### PR TITLE
Add error message on failed builds

### PR DESCRIPTION
**Pull request by gregone**
Screen Shot 2021-04-07 at 19 33 55

Fixes for https://github.com/hashicorp/waypoint/issues/1266

### CODE REVIEW

**Review by mitchellh — CHANGES_REQUESTED** [review]
I think we need this same behavior for all operations? Deploys and releases.

Looks great though.

File: .changelog/1280.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+ui/build-logs: output failed build errors in logs
```
Comment (mitchellh) [inline .changelog/1280.txt:2]: I think UI can just have a `ui:` prefix

**Comment by gregone** [comment]
@mitchellh So the `operation-logs` component is shared and used in the build, deployment, and releases templates, so that's covered, But i'll add the output in the badge to the other pages now

**Comment by gregone** [comment]
@mitchellh added in https://github.com/hashicorp/waypoint/pull/1280/commits/ad7265b4b865020fd7c3f65e7e2d5835428e9100

**Review by mitchellh — APPROVED** [review]

**Pull request merged**